ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PurchaseButton.kt

```diff
@@ -41,6 +42,8 @@ import com.revenuecat.purchases.ui.revenuecatui.data.testdata.TestData
 import com.revenuecat.purchases.ui.revenuecatui.extensions.introEligibility
 import com.revenuecat.purchases.ui.revenuecatui.helpers.LocalActivity
 
+internal const val PURCHASE_BUTTON_TAG = "PurchaseButton"
```

Comment:
This is adding several new global values / methods.
What do you think about adding a type that group all these? Something like `TestTag.purchaseButton`, `TestTag.selectButton(packageId:)`, etc.